parsl/tests/test_utils/test_execute_wait.py

```diff
@@ -0,0 +1,35 @@
+import pytest
+
+from parsl.utils import execute_wait
+
+
+@pytest.mark.local
+def test_env():
+    ''' Regression testing for issue #27
+    '''
+
+    rc, stdout, stderr = execute_wait("env", 1)
+
+    stdout = stdout.split('\n')
+    x = [s for s in stdout if s.startswith("PATH=")]
+    assert x, "PATH not found"
+
+    x = [s for s in stdout if s.startswith("HOME=")]
+    assert x, "HOME not found"
```

Comment:
I get this is just moving the test, but I also don't understand what it's testing.  What does this have to do with the referenced issue #27, which looks related to documentation?

Meanwhile, `PATH/HOME not found` is restating the logic.  Is the goal to just ensure we're executing something successfully?  If so, something like `Did not execute: canary value missing!` might be a better assertion message.  Or is the environment actually important to verify?  I don't see any reference to a special environment in `execute_wait` ... ?

And of course, a simplification of the test itself might be:
```python
assert any(s.startswith("PATH=") for s in stdout), "Bad environment!"
...
```

Reply:
The #27 is a red herring -- around 2017 there was a somewhat misguided activity to separate out the provider layer into its own github repo called `libsubmit`. This refers to libsubmit issue 27: https://github.com/Parsl/libsubmit/issues/27. 

Go back far enough in history and the commit that introduces this test does stuff in the libsubmit/ subdirectory from the subsequently re-merged libsubmit history.

I'm not going to tidy these tests any time soon - this PR sequence really is meant to be moving, deleting and then moving on.